comparisons/CompareGraphs.py

```diff
@@ -83,7 +86,13 @@
         hToCompare[iFile].SetStats(0)
     else:
         hToCompare[iFile].SetName(f'g{iFile}')
-    SetObjectStyle(hToCompare[iFile], color=GetROOTColor(color), markerstyle=GetROOTMarker(marker), fillstyle=0)
+    SetObjectStyle(hToCompare[iFile],
+                   color=GetROOTColor(color),
+                   markerstyle=GetROOTMarker(marker),
+                   markersize=inputCfg['options']['markersize'],
+                   linewidth=inputCfg['options']['linewidth'],
+                   fillstyle=inputCfg['options']['fillstyle'][iFile], 
+                   fillalpha=inputCfg['options']['fillalpha'][iFile])
```

Comment:
I would access the dictionary only once defining the variables as done for the others

Reply:
Hi @fcatalan92! Now it should be ok. Let me know if there is still something wrong!